plugin/src/main/groovy/com/here/gradle/plugins/jobdsl/tasks/runners/AbstractTaskRunner.groovy

```diff
@@ -79,6 +79,8 @@ abstract class AbstractTaskRunner {
             it.level = Level.SEVERE
         }
 
+        jenkinsRule.timeout = 1800 // Seconds
```

Comment:
I would prefer to disable the timeout by setting it to 0 instead. The plugin should not enforce any timeout for the user.

Reply:
Modified to 0, thanks for the quick review!